File: Adventure4YouAPI/ViewModels/Points/PointDetailViewModel.cs
using System;

namespace Adventure4YouAPI.ViewModels.Points
{
    public class PointDetailViewModel
    {
        public Guid PointId { get; set; }
        public Guid StageId { get; set; }
        public string Name { get; set; }
        public PointTypeViewModel Type { get; set; }
        public int Value { get; set; }
        public double Latitude { get; set; }
        public double Longitude { get; set; }
        public string Answer { get; set; }
        public string Message { get; set; }
    }
}

File: Presentation/src/app/races/shared/models/point-detail-view-model.ts
import { PointType } from './point-type';

export class PointDetailViewModel {
    public pointId: string;
    public stageId: string;
    public name: string;
    public type: PointType;
    public value: number;
    public latitude: number;
    public longitude: number;
    public answer: string;
    public message: string;
}
